### ISSUE TITLE
Retina net in official\vision\detection\main.py does not get desired accuracy 

### ISSUE THREAD

**Issue by myopengit** [tensorflow/models#9843]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

I am using the latest TensorFlow Model Garden release and TensorFlow 2.
I am reporting the issue to the correct repository. (Model Garden official or research directory)
I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/...

## 2. Describe the bug

The loss of retinanet doesn't go down (total loss stays between 0.6 to 1.2), default setting is used for training and the evaluation accuracy after 22500 steps is just 0.06

## 3. Steps to reproduce

TPU_NAME="<your GCP TPU name>"
MODEL_DIR="<path to the directory to store model files>"
RESNET_CHECKPOINT="<path to the pre-trained Resnet-50 checkpoint>"
TRAIN_FILE_PATTERN="<path to the TFRecord training data>"
EVAL_FILE_PATTERN="<path to the TFRecord validation data>"
VAL_JSON_FILE="<path to the validation annotation JSON file>"
python3 ~/models/official/vision/detection/main.py \
  --strategy_type=tpu \
  --tpu="${TPU_NAME?}" \
  --model_dir="${MODEL_DIR?}" \
  --mode=train \
  --params_override="{ type: retinanet, train: { checkpoint: { path: ${RESNET_CHECKPOINT?}, prefix: resnet50/ }, train_file_pattern: ${TRAIN_FILE_PATTERN?} }, eval: { val_json_file: ${VAL_JSON_FILE?}, eval_file_pattern: ${EVAL_FILE_PATTERN?} } }"

## 4. Expected behavior

give the correct accuracy 

## 6. System information

- OS Platform and Distribution (GCP VM TF2.4.1, TPU 2.4.1):

**Comment by Srikeshram** [comment]
It indicates that the model is **Underfitting**. It is caused due to 

- Not enough training data .
- Relationship between data is unlearnable.

if there is no presence of above issues then it may be solve by 

- Reducing the size of the input image .
- Reduce the rate regularization or dropout .
- Adding more Dense or CNN layers.

**Comment by yeqingli** [comment]
hi, which evaluation accuracy are you referring to? Do you have a screenshot of the tensorboard or log of accuracy?
Plus, are you using the RESNET_CHECKPOINT and what is the original url?

**Comment by myopengit** [comment]
Thank you for your reply:

@yeqingli RESNET_CHECKPOINT is model-garden-vision_detection_resnet50-2018-02-07.tar.gz, downloaded from the TensorFlow official website https://github.com/tensorflow/models/tree/master/official/vision/detection.  I obtained the accuracy using the following command:
python3 ~/models/official/vision/detection/main.py --strategy_type=tpu --tpu="***" --model_dir="***" --mode=eval --params_override="{eval: { val_json_file: ***, eval_file_pattern: ***.tfrecord, eval_samples: 5000 } }"

Training details:
@Srikeshram I used the coco dataset and the default setting for training, Below are the training commands:
nohup python3 ~/models/official/vision/detection/main.py --strategy_type=tpu --tpu="***" --model_dir="gs://***/" --mode=train --params_override="{ type: retinanet, train: { checkpoint: { path: gs://***/retinanet, prefix: resnet50/ }, train_file_pattern: gs://***/dataset/coco2017/train1/train-00000-of-00001.tfrecord}, eval: { val_json_file: gs://***/dataset/coco2017/instances_val2017.json, eval_file_pattern: gs://***/dataset/coco2017/val/val-00000-of-00001.tfrecord} }" &

The training log can be found in the following link:
https://drive.google.com/file/d/1PdMufQLiPA2t3aLIIcgv0aYZFKleiIsJ/view?usp=sharing
